mu/wheels/__init__.py

```diff
@@ -41,22 +41,21 @@ class WheelsBuildError(WheelsError):
     # pygame versions >=2.1.3 have issues in macOS 10.x, so temporarily for
     # Mu release 1.2.1 pin the max version here
     # https://github.com/mu-editor/mu/issues/2423
-    ("pygame", "pygame<2.1.3"),
-    ("numpy", "numpy"),
-    ("pgzero", "pgzero>=1.2.1", "--no-dependencies"),
-    # Flask v1 depends on Jinja v2, which doesn't have an upper bound limit in
-    # MarkupSafe, and v2.1 is not compatible with Jinja v2
-    ("flask", "flask==2.0.3"),
+    ("pygame", ("pygame<2.1.3",)),
+    ("numpy", ("numpy",)),
+    ("pgzero", ("pgzero>=1.2.1",), "--no-dependencies"),

```

Comment:
The pygame entry here was added here to ensure we installed a known version (pgzero installs the latest). Numpy was only added because it's needed as the only other pgzero dependency, since `--no-dependencies` doesn't pull them.

So I believe with this PR we can also change this line to this? (haven't test it):
```suggestion
    ("pgzero", ("pgzero>=1.2.1", "pygame<2.1.3")),
```

Reply:
Thanks @carlosperate, I'll give it a go... :)